contrib/linux/install-icons.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
```

Comment:
For the header, check the .sh files in the ./scripts area.
(They have an SPDX, short description, and usage).

Reply:
Suggest using `set -e` at the top to quit on the first error. 
Or if using bash, `set -euo pipefile` to quit on the first error, and undefined variables, and also any inter-pipe failures.

Reply:
Added the header and the `set -e`.  I suppose I'll try to stay POSIX compliant in order to support the odd BSD user.  I believe most of the desktop stuff is going to be the same on BSD as it is on Linux.